bokehjs/src/lib/models/renderers/glyph_renderer.ts

```diff
@@ -364,7 +366,11 @@ export class GlyphRendererView extends DataRendererView {
           this.hover_glyph.render(ctx, this.model.view.convert_indices_from_subset(inspected_subset_indices))
         else
           glyph.render(ctx, all_indices)
-      } else if (this.glyph instanceof PatchView || this.glyph instanceof HAreaView || this.glyph instanceof VAreaView) {
+        } else if (this.glyph instanceof PatchView ||
```

Comment:
This needs to be shifted back to clear the lint error.

Reply:
Right, sorry for missing that! Fixed now.

@bryevdv No worries, I'm not in a hurry, just wanted to make sure that there's nothing pending from my end.